packages/functionals/botpress-qna/src/index.js

```diff
@@ -70,5 +74,67 @@ module.exports = {
         res.status(500).send(e.message || 'Error')
       }
     })
+
+    router.get('/csv', async (req, res) => {
+      const questions = await storage.getQuestions()
+      const rows = questions.flatMap(item => {
```

Comment:
do you know since which version Node.js supports `flatMap`?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap says it's not supported but obviously it worked for you. or is it polyfilled in babel-core?

Reply:
Nice tool for node https://node.green